tests/system/action/point_of_order_category/test_create.py

```diff
@@ -1,7 +1,11 @@
+from openslides_backend.permissions.permissions import Permissions
 from tests.system.action.base import BaseActionTestCase
 
 
 class PointOfOrderCategoryCreate(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+
```

Comment:
remove

Reply:
Done, was there nothing else? Something big?

Reply:
Not from me, rest looks good. Maybe @r-peschke has some more remarks?